src/Utils/Helpers.ts

```diff
@@ -36,15 +36,35 @@ export class RealPath {
     this.absPath = absPath;
   }
 
-  public static isEqual(path0: RealPath, path1: RealPath): boolean {
-    return path0.absPath === path1.absPath;
+  public static areEqual(path0: string, path1: string): boolean {
+    const realPath0 = this.createRealPath(path0);
+    const realPath1 = this.createRealPath(path1);
+
+    if (!realPath0 || !realPath1) {
+      return false;
+    }
+
+    return realPath0.equal(realPath1);
+  }
+
+  public equal(lhs: RealPath): boolean {
+    return this.absPath === lhs.absPath;
   }
 
   public static createRealPath(rawPath: string): RealPath|null {
     const absPath = path.resolve(path.normalize(rawPath));
 
     return fs.existsSync(absPath) ? new RealPath(absPath) : null;
   }
+
+  public static exist(rawPath: string|undefined): boolean {
+    if (!rawPath) {
+      return false;
+    }
```

Comment:
IMHO for the case of undefined rawPath, how about returning the `this.abspath`'s existence? When I read this code, I was expected to use `exist` method with a `RealPath` instance, for example 
```ts
filepath = RealPath('~/my_file.cfg');
filepath.exist()
```

Reply:
It's not a member function, but a static function.

Usage: 
```
RealPath.exist('~/my_file.cfg');
```

Reply:
😮 Now I catch it..!

Reply:
Now I catch it..! 😮